modules/80-printers.yml

```diff
@@ -15,3 +15,12 @@ source:
   # Config
   - system-config-printer-common
   - system-config-printer-udev
+modules:
+- name: hplip
+  type: shell
+  commands:
+  - apt install -y jq
+  - mkdir -p /tmp/hplip
```

Comment:
I'm guessing you have to also cd into that directory before downloading with wget